docs/line.asciidoc

```diff
@@ -27,9 +27,27 @@ NOTE: In Elasticsearch releases 1.4.3 and later, this functionality requires you
 {ref}/modules-scripting.html[dynamic Groovy scripting].
 
 The availability of these options varies depending on the aggregation you choose.
-Select *view options* to change the following aspects of the table:
 
+Select the *Options* tab to change the following aspects of the chart:
+
+*Y-Axis Scale*:: You can select *linear*, *log*, or *square root* scales for the chart's Y axis.
```

Comment:
Can you give examples of where each of these might be useful? A bit of material: http://www.forbes.com/sites/naomirobbins/2012/01/19/when-should-i-use-logarithmic-scales-in-my-charts-and-graphs/